python/chiquito/dsl.py

```diff
@@ -23,6 +22,7 @@ def __init__(self: Circuit):
         self.ast = ASTCircuit()
         self.witness = TraceWitness()
         self.rust_ast_id = 0
+        self.num_step_instances = 0
```

Comment:
@qwang98 it seems we don't need this field anymore

Reply:
Surprises me why I didn't remove it already but fixed and thanks for catching. :)